package.json

```diff
@@ -26,12 +26,14 @@
     "lang:refresh": "node scripts/refresh-language.js",
     "csl:refresh": "node scripts/update-csl.js",
     "reveal:build": "node scripts/update-reveal.js",
-    "wp:dev": "cross-env NODE_ENV=development webpack",
-    "wp:prod": "cross-env NODE_ENV=production webpack",
-    "watch": "gulp watch",
+    "wp:dev": "cross-env NODE_ENV=development webpack --progress --colors",
+    "wp:prod": "cross-env NODE_ENV=production webpack --progress --colors",
+    "watch:gulp": "gulp watch",
+    "watch:wp": "cross-env NODE_ENV=development webpack --progress --colors --watch",
+    "watch": "npm-run-all --parallel watch:*",
```

Comment:
What exactly does this command do? I assume simply run all in parallel processes…? If so, can't we emulate that behaviour either by merging them in a single watch command in the gulp-file or by using Yarn? (Just to only have yarn in here, and not npm…?)

Reply:
Yes, it runs them in parallel. Although it has npm in it's name, it's completely compatible with yarn (https://www.npmjs.com/package/npm-run-all#yarn-compatibility). Thus `yarn watch` is equivalent to running `yarn watch:wp` and `yarn watch:gulp` in parallel.

Reply:
Perfect, thanks!